tests/integration/__tests__/patterns-dirs.js

```diff
@@ -224,11 +224,61 @@ describe("Ignore symlinks", () => {
   });
 
   testPatterns("", ["test-a/*"], { stdout: "test-a/a.js\n" }, base);
-  testPatterns("", ["test-a/symlink-to-directory-a"], { stdout: "???" }, base);
-  testPatterns("", ["test-a/symlink-to-directory-b"], { stdout: "???" }, base);
-  testPatterns("", ["test-a/symlink-to-file-a"], { stdout: "???" }, base);
-  testPatterns("", ["test-a/symlink-to-file-b"], { stdout: "???" }, base);
-  testPatterns("", ["test-a/symlink-*"], { stdout: "???" }, base);
+  testPatterns(
+    "",
+    ["test-a/symlink-to-directory-a"],
+    {
+      status: 2,
+      stdout: "",
+      stderr:
+        "[error] Explicitly specified symbolic link was ignored: test-a/symlink-to-directory-a\n",
+    },
+    base
+  );
+  testPatterns(
+    "",
+    ["test-a/symlink-to-directory-b"],
+    {
+      status: 2,
+      stdout: "",
+      stderr:
+        "[error] Explicitly specified symbolic link was ignored: test-a/symlink-to-directory-b\n",
+    },
+    base
+  );
+  testPatterns(
+    "",
+    ["test-a/symlink-to-file-a"],
+    {
+      status: 2,
+      stdout: "",
+      stderr:
+        "[error] Explicitly specified symbolic link was ignored: test-a/symlink-to-file-a\n",
+    },
+    base
+  );
+  testPatterns(
+    "",
+    ["test-a/symlink-to-file-b"],
+    {
+      status: 2,
+      stdout: "",
+      stderr:
+        "[error] Explicitly specified symbolic link was ignored: test-a/symlink-to-file-b\n",
+    },
+    base
+  );
+  testPatterns(
+    "",
+    ["test-a/symlink-*"],
+    {
+      status: 2,
+      stdout: "",
+      stderr:
+        '[error] No files matching the pattern were found: "test-a/symlink-*".\n',
+    },
+    base
+  );
```

Comment:
What's the result of `prettier test-a/symlink-to-directory-b/*`, let's test it too. (Not really care about the result)

Reply:
Added. That actually succeeds, which is perhaps surprising, but `fast-glob` [documents](https://www.npmjs.com/package/fast-glob#followsymboliclinks) that the `followSymbolicLinks` option does not affect the base directory of the pattern.